packages/vats/src/core/basic-behaviors.js

```diff
@@ -696,11 +700,13 @@ export const BASIC_BOOTSTRAP_PERMITS = {
       bridgeManager: 'bridge',
       // TODO: re-org loadCriticalVat to be subject to permits
       loadCriticalVat: true,
+      startUpgradable: true,
       zoe: true,
     },
     produce: {
       bankManager: 'bank',
       bldIssuerKit: true,
+      bldMintHolderKit: true,
```

Comment:
This requires an update to the bootstrap visualization snapshot.

In `packages/vats` run `yarn test test/test-boot-config.js --update-snapshots`

Reply:
yep. found it.